environment_test.yml

```diff
@@ -36,3 +36,10 @@ dependencies:
   - pyvista
   - pip
   - python-kaleido
+  # Linters and code style
+  - black
```

Comment:
make sure that this version matches the current version in the .pre-commit.yml file. (I think 21.7b0)